src/arch/xtensa/include/arch/schedule/task.h

```diff
@@ -2,76 +2,58 @@
  *
  * Copyright(c) 2017 Intel Corporation. All rights reserved.
  *
- * Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
+ * Author: Tomasz Lauda <tomasz.lauda@linux.intel.com>
  */
 
```

Comment:
I am inclined to say that we never remove Authors, we just new entries.

Reply:
Fixed.